src/components/date-picker/z-date-picker/index.stories.ts

```diff
@@ -59,32 +60,42 @@ export const Date: Story = {
   args: {
     datePickerId: "picker-01",
     label: "ZDatePicker with date",
+    value: "2021-01-01",
+    name: "date-picker",
```

Comment:
i `name` si possono anche togliere dagli args dato che non vengono passati al componente (non ha neanche granché senso provarli da storybook in effetti)